nextcord/reaction.py

```diff
@@ -2,18 +2,42 @@
 
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Optional, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Union
 
 from .iterators import ReactionIterator
 
 __all__ = ("Reaction",)
 
 if TYPE_CHECKING:
     from .abc import Snowflake
+    from .colour import Color
```

Comment:
From a glance, looks like this is in type checking but used below for `burst_colors` (don't we use british english). For that anyway, it could probably be a (cached) property to save memory for the majority of people that do not want the burst reaction colours.

Reply:
Oh okay thanks, will do that
Should I replace `burst_colors` with colours to make an alias?